### PR TITLE
Assorted small bugfixes

### PR DESCRIPTION
**Pull request by lwlshawn**
Resolves the following:
* #198 : "X persons listed" message -> "X clients listed"
* #197 : "1 orders/tasks/clients listed" -> "1 order(s)/task(s)/client(s) listed"
* #190 : Update UG to make it clear that two people with same name are considered same person
* #189 : Limit phone number field to at most 20 digits and update error message
* #178 : "listed all Xs" -> "listed all X(s)" for listX commands
* #175 : Fixed typon UG regarding marktask command

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -264,7 +270,7 @@ Format: `marktask INDEX`
 * The index **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `listtask` followed by `deletetask 2` deletes the 2nd client in the application.
+* `listtask` followed by `marktask 2` deletes the 2nd client in the application.
```
Comment (ngchisern) [inline docs/UserGuide.md:273]: `marks the 2nd task as done`

**Review by ngchisern — CHANGES_REQUESTED** [review]
Looks good. Just need to fix one minor issue.
Are you supposed to change `persons` to `clients` in the feedback messages of other commands related to persons in this pr too?

**Comment by lwlshawn** [comment]
Have fixed the raised issues; and additionally made the update to UG help image, resolving #176

**Review by ngchisern — APPROVED** [review]
LGTM

**Pull request merged**